configs/katello/4.4.yaml

```diff
@@ -0,0 +1,146 @@
+---
+:project: katello
+:github_org: katello
+:releases:
+  :4.4.0:
+    :redmine_version_id: 1517
+:prior_releases:
+  :4.3.1:
+    :redmine_version_id: 1535
+  :4.3.0:
```

Comment:
4.3.0 and 4.2.2 are already GA so there's no need to count them as priors (ie nothing will be cherry picked into or from those releases now)